examples/bridge-app/esp32/main/CHIPDeviceManager.cpp

```diff
@@ -79,23 +79,16 @@ CHIP_ERROR CHIPDeviceManager::Init(CHIPDeviceManagerCallbacks * cb)
         ConnectivityMgr().SetWiFiAPMode(ConnectivityManager::kWiFiAPMode_Enabled);
     }
 
-    err = Platform::MemoryInit();
-    if (err != CHIP_NO_ERROR)
-    {
-        return err;
-    }
-
     // Register a function to receive events from the CHIP device layer.  Note that calls to
     // this function will happen on the CHIP event loop thread, not the app_main thread.
     PlatformMgr().AddEventHandler(CHIPDeviceManager::CommonDeviceEventHandler, reinterpret_cast<intptr_t>(cb));
 
     // Start a task to run the CHIP Device event loop.
     err = PlatformMgr().StartEventLoopTask();
-    if (err != CHIP_NO_ERROR)
-    {
-        return err;
-    }
-    return CHIP_NO_ERROR;
+    SuccessOrExit(err);
```

Comment:
Please prefer return macros over `goto exit` patterns.

use `ReturnErrorOnFailure` or even more directly: `return PlatformMgr().StartEventLoopTask()`

Reply:
This is probably best done as a followup that changes the whole function...  I don't think we should block this fix on that work, just like we did not block https://github.com/project-chip/connectedhomeip/pull/15508 (which had the same pattern).

Reply:
@chshu Addressed this in https://github.com/project-chip/connectedhomeip/pull/15553

Reply:
I agree, however my complaint was: this method used to use 'return' and it was converted to 'goto exit' and that does not seem right.

Reply:
Ah, I see.  Sorry, I missed that...

In any case, addressed in #15553.